account-kit/react/src/components/auth/card/eoa.tsx

```diff
@@ -153,30 +153,34 @@ export const EoaPickCard = () => {
 
   const { walletConnectParams } = useWalletConnectAuthConfig();
 
-  const connectorButtons = connectors.map((connector) => {
-    return (
-      <Button
-        className="justify-start"
-        variant="social"
-        key={connector.id}
-        icon={
-          connector.icon && (
-            <img
-              src={connector.icon}
-              alt={connector.name}
-              height={20}
-              width={20}
-            />
-          )
-        }
-        onClick={() => {
-          connect({ connector, chainId: chain.id });
-        }}
-      >
-        {connector.name}
-      </Button>
-    );
-  });
+  const connectorButtons = connectors
+    .filter((x) => x.type !== "walletConnect")
```

Comment:
I didn't want to have to change everything thing, so I'm just filtering this out and rendering how we used to below